modules/platforms/dotnet/Apache.Ignite.Core.Tests/Client/Cache/ListLogger.cs

```diff
@@ -21,45 +21,126 @@ namespace Apache.Ignite.Core.Tests.Client.Cache
     using System.Linq;
     using Apache.Ignite.Core.Log;
 
+    /// <summary>
+    /// Stores log entries in a list.
+    /// </summary>
     public class ListLogger : ILogger
     {
         /** */
-        private readonly List<string> _messages = new List<string>();
+        private readonly List<Entry> _entries = new List<Entry>();
 
         /** */
         private readonly object _lock = new object();
 
-        public List<string> Messages
+        /** */
+        private readonly ILogger _wrappedLogger;
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="ListLogger"/> class.
+        /// </summary>
+        public ListLogger(ILogger wrappedLogger = null)
+        {
+            _wrappedLogger = wrappedLogger;
+        }
+
+        /// <summary>
+        /// Gets the entries.
+        /// </summary>
+        public List<Entry> Entries
```

Comment:
Could we make this getter an Entry class internal?

Reply:
Please explain in more detail - how can we move this inside Entry class?